### DISCUSSION TITLE
Demo ufficialmente accessibile

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#586] [Idee e proposte]
Buonasera volevo dire che la demo è ufficialmente accessibile
per le credenziali scrivete: fcoco953@gmail.com
Link registro: Registro2324.altervista.org/public
link gifthub https://github.com/Davide031/Demo-Giua-school/

**Comment by Mattias123A** [comment]
Ho contattato f coco ma nessuna risposta. Sapete dirmi le credenziali? Grazie ancora
